ckanext/validation/logic.py

```diff
@@ -683,7 +690,7 @@ def _run_sync_validation(resource_id, local_upload=False, new_resource=True):
 
             raise t.ValidationError({
                 u'validation': [report]})
-    else:
-        raise t.ValidationError({
-            'validation': []
-        })
+        else:
+            raise t.ValidationError({
+                'validation': []
+            })
```

Comment:
Take a look at my previous comment to line :669
Indenting this else block in breaks the earlier logic. Before a validation error was risen if no validation report avaiable. Right now nothing happens in this case.